README.md

```diff
@@ -68,7 +69,7 @@ docker run \
   tooljet/try:EE-LTS-latest
 ```
 
-*For users upgrading their ToolJet version, we recommend choosing the LTS version over the latest version. The LTS version ensures stability with production bug fixes, security patches, and performance enhancements.*
+_For users upgrading their ToolJet version, we recommend choosing the LTS version over the latest version. The LTS version ensures stability with production bug fixes, security patches, and performance enhancements._
```

Comment:
Hey @prathamranjansingh!
Can you please revert this change? Use ** instead of _ _ for this line.